### DISCUSSION TITLE
Zwei OpenDTU OnBattery welcher Wechselrichter wird bevorzugt?

### DISCUSSION THREAD

**Discussion by GTOZP** [hoylabs/OpenDTU-OnBattery#815] [General]
Hallo.. ich habe zwei identisch eingestellte OpenDTU OnBattery die jeweils einen Hoymiles HM-1200 ansteuern. Die Hoymiles hängen jeweils nach einer Zendure Solarflow. Es wird immer der (nennen wir ihn Hoy_1) eine Wechselrichter "bevorzugt", sprich er bekommt bei gleicher Limitvorgabe in OnBattery immer die höhere Anforderung zur Leistungsabgabe. Ich möchte aber dass Hoy_2 (der andere WR) bevorzugt angesprochen wird da an diesem System die Solarmodule mit mehr Sonneneinstrahlung hängen. Den einzigen Unterschied was ich bisher bemerkte war die niedrigere IP-Adresse des Hoy_1
Kann man das irgendwie beeinflussen? Danke!

**Comment by spcqike** [comment]
Hallo 

das kannst du nicht direkt beeinflussen.  Du hast zwei Systeme die autark laufen. Jedes System hat ein Update interval. Das System, welches zuerst läuft und aktualisiert, führt. So einfach. 

Du kannst versuchen dein sys_1 Träger zu machen. Also die update Intervalle zu erhöhen. Das wurde hier bereits mehrfach diskutiert und vorgestellt. 

Eine weitere Möglichkeit wären unterschiedliche power Limits und targets. In der Hoffnung, dass das PV System mehr drückt. 

Bspw das PV System auf ein Target von -10W. Das batteriesystem auf +10W. Das könnte dazu führen, dass das PV System am Ende -10W erreicht, das batteriesystem sieht „oh, -10W, ich speise 20W zu viel ein“ und sich dann zurück fährt. 

Ggf auch eine Mischung aus beidem. Beim batteriesystem das interval reduzieren und das Limit erhöhen. 

Egal wie, es sind alles nur halbe Lösungen. Die Systeme sind autark und kennen sich nicht. Eine gezielte Steuerung gibt es also nicht und es kann und wird immer zu ungewollten aufgeschwungenen Zuständen kommen.
